src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const declarations = sourceString.split(';');
```

Comment:
When splitting the 'sourceString' by semicolons, there might be empty entries due to consecutive semicolons. Consider filtering out empty strings immediately after the split operation to simplify the subsequent forEach loop.